### PR TITLE
[WIP] Fix broken symlinks

### PR DESCRIPTION
**Pull request by chaen**
Addresses part of https://github.com/DIRACGrid/DIRACOS/issues/46

BEGINRELEASENOTES
NEW: test that there are no absolute symlinks in DIRACOS
FIX: symlinks are transformed into copy by tar
FIX: create a virtualenv with no external symlinks
ENDRELEASENOTES

### CODE REVIEW

File: tests/integration/test_symlink.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# ./usr/lib64/python2.7/os.py -> /usr/lib64/python2.7/os.py
+
+# We also replace path starting with $DIRACOS to './'
+
+find $DIRACOS -type l -exec ls -l {} \; | sed "s|-> $DIRACOS|-> ./|g" | grep '\-> /'
+
+# If grep returns 0, some items were found -> not good
+if [ $? -eq 0 ];
+then
+   echo "Some absolute symlinks were found";
```
Comment (petricm) [inline tests/integration/test_symlink.sh]: I would add here an output of the absolute symlinks if they are present

Comment (chaen) [reply tests/integration/test_symlink.sh]: They appear, above. That's why I did not use `grep -c`

File: diracos/bundlelib.py [hunk 2]
```diff
@@ -185,8 +185,40 @@ def _resolveAllPackageDependencyURLs(requiredPkg=None, ignoredPackages=None):
...
+    if [ ! -f $fp ];
+    then
+      # We display the link, but replace the actual DIRACOS path with just 'DIRACOS'
+      echo -n "$i\n" | sed "s|^$DIRACOS|DIRACOS/|g";
+    fi;
+  fi;
+done)
+
+# The file MUST be sorted for comparison, and we remove the empty lines
+echo -e $brokenLinks | sort | sed '/^$/d' > /tmp/diracos/brokenLinks.txt
```
Comment (petricm) [inline diracos/bundlelib.py]: Here you have `/tmp/diracos/brokenLinks.txt` hardcoded, it should be `$DIRACOS/brokenLinks.txt`

Comment (chaen) [reply diracos/bundlelib.py]: In the bundeling step, `DIRACOS` is `/tmp/diracos`. But you're right, I will replace all that with a variable, it's cleanier

**Pull request closed without merging**